src/rdkafka_conf.h

```diff
@@ -98,7 +98,10 @@ typedef	enum {
 	_RK_CONSUMER = 0x4,
 	_RK_TOPIC = 0x8,
         _RK_CGRP = 0x10,
-        _RK_DEPRECATED = 0x20
+        _RK_DEPRECATED = 0x20,
+        _RK_HIDDEN = 0x40,
+        _RK_HIGH = 0x80, /* High Importance */
+        _RK_MED = 0x100  /* Medium Importance */
```

Comment:
Should there be a `Low Importance`?  Surely there are a few configuration properties most people needn't touch.  For example `message.copy.max.bytes` is not really something most people will ever need to tune.

Reply:
The default is low (see generated CONFIGURATION.md).